glowbot/tasks.py

```diff
@@ -52,6 +52,14 @@ def is_now(start, end, now):
 
             if not is_now(seeding_start_time, seeding_end_time, time_now):
                 self.logger.debug(f'Not within seeding time range of \"{seeding_start_time_str} - {seeding_end_time_str}\" UTC')
+                status_string = "Outside seeding window"
+                # presence updates rate limited to 5 updates / 20s:
+                if self.bot.ws is not None:
+                    # websocket may not be created first time this is run. it is created during bot.run().
+                    # i don't think update_seeders.start() should be run from __init__()...
+                    # but from where? discord.on_ready() seems promising, but documenation
+                    # claims it could be called more than once.
+                    await self.bot.change_presence(status=discord.Status.idle, activity=discord.Game(status_string))
```

Comment:
While I appreciate this as a feature, and definitely want this, this only works in a single-server set up in this current form.  In a testing case where two RCON endpoints are being watched, it will only ever report the seeding status of the second one.  I'd ask we break this out into a separate PR if at all possible and maybe we need to update the statuses in a different way to reflect more than one server.

Reply:
I think this can be pretty easily adapted TBH, so don't stress about it too much.  It's real slick tho 👀